app/javascript/features/form/components/DialogContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Button
+} from "@material-ui/core";
+import React from "react";
+
+type OpenDialog = {
+  isOpenSuccessDialog: boolean;
```

Comment:
```suggestion
  isOpen: boolean;
```
isOpenくらいシンプルでも良いと思う